DEPTH_COLOR_TOPIC: /camera/color/image_raw
DEPTH_REGISTERED_TOPIC: /camera/depth_registered/points
# 원래는 아래와 같은 토픽명을 가져야 함
# DEPTH_COLOR_TOPIC: /depth_01/color/image_raw
# DEPTH_REGISTERED_TOPIC: /depth_01/depth_registered/points


DEPTH_IMAGE_01_TOPIC: /depth_01_image_01
DEPTH_IMAGE_02_TOPIC: /depth_01_image_02

DEPTH_CAMERA_POINTCLOUD_TOPIC: /depth_01_camera_pointcloud
DEPTH_WORLD_POINTCLOUD_TOPIC: /depth_01_world_pointcloud

DEPTH_TF_TOPIC: depth_01_tf

# Chessboard 규격에 대한 파라미터
CHESSBOARD_EDGE_SIZE: 0.1003
CHESSBOARD_WIDTH: 9
CHESSBOARD_HEIGHT: 7

# 카메라 프레임에 대한 파라미터
FRAMES_PER_SECONDS: 10